Documentation/format/index.md

```diff
@@ -0,0 +1,198 @@
+
+The section contains a sequence of the raw string data, each prefixed with the string's length.
+Strings are referenced by pointing to the beginning of their length field. The strings are sorted in lexicographically ascending order.
+
+```
+┌─────────────────────────┬───────────────┐
+│ count(symbols) <4 byte> │ len <4 byte>  │
+├─────────────────────────┴───────────────┤
+│ ┌─────────────────────┬───────────────┐ │
+│ │ len(str_1) <varint> │ str_1 <bytes> │ │
+│ ├─────────────────────┴───────────────┤ │
+│ │                . . .                │ │
+│ ├─────────────────────┬───────────────┤ │
+│ │ len(str_n) <varint> │ str_1 <bytes> │ │
+│ └─────────────────────┴───────────────┘ │
+├─────────────────────────────────────────┤
+│  CRC32 <4 byte>                         │
+└─────────────────────────────────────────┘
+```
+
+
+### Series
+
+The section contains a sequence of series that hold the label set of the series as well as the chunks within the block. The series are sorted lexicographically by their label sets.  
+The file offset to the beginning of a series serves as the series' ID in all subsequent references. Thereby, a sorted list of series IDs implies a lexicographically sorted list of series label sets.
+
+```
+┌───────────────────────────────────────┐
+│  count(series) <4 byte>               │
+├───────────────────────────────────────┤
+│ ┌───────────────────────────────────┐ │
+│ │   series_1                        │ │
+│ ├───────────────────────────────────┤ │
+│ │                 . . .             │ │
+│ ├───────────────────────────────────┤ │
+│ │   series_n                        │ │
+│ └───────────────────────────────────┘ │
+└───────────────────────────────────────┘
+```
+
+Every series holds a list of label pairs and chunks. The label pairs reference the symbol table and the chunks an address in one of the block's chunk files.
+
+```
+┌─────────────────────────────────────────────────────────┐
+│ len <varint>                                            │
+├─────────────────────────────────────────────────────────┤
+│ ┌──────────────────┬──────────────────────────────────┐ │
+│ │                  │ ┌──────────────────────────┐     │ │
+│ │                  │ │ ref(l_i.name) <varint>   │     │ │
+│ │ #labels <varint> │ ├──────────────────────────┤ ... │ │
+│ │                  │ │ ref(l_i.value) <varint>  │     │ │
+│ │                  │ └──────────────────────────┘     │ │
+│ ├──────────────────┼──────────────────────────────────┤ │
+│ │                  │ ┌──────────────────────────┐     │ │
+│ │                  │ │ c_i.mint <varint>        │     │ │
+│ │                  │ ├──────────────────────────┤     │ │
+│ │                  │ │ c_i.maxt <varint>        │     │ │
+│ │ #chunks <varint> │ ├──────────────────────────┤ ... │ │
+│ │                  │ │ ref(c_i.data) <varint>   │     │ │
+│ │                  │ ├──────────────────────────┤     │ │
+│ │                  │ │ crc32(c_i.data) <varint> │     │ │
```

Comment:
Yea, but then we would just use 4 or 8 byte ints to begin with. The whole point here is compression. With the known tradeoff of performance. But this shouldn't matter here.

Our chunk compression obviously does the same and that at a much higher cardinality.